versions.tf

```diff
@@ -11,5 +11,13 @@ terraform {
       source  = "hashicorp/random"
       version = ">= 3.0.0"
     }
+
+    # TODO(luke): uncomment this in a follow up commit to make terraform-aws-collection
+    # more likley to avoid the terraform apply failure
+    # (due to s3 bucket notification eventual consistency)
+    time = {
```

Comment:
Seems like this is out of sync with the rest? Meaning if this provider isn't used, the requirement should be commented out?

Reply:
done